R/RcppExports.R

```diff
@@ -1,14 +1,22 @@
 # Generated by using Rcpp::compileAttributes() -> do not edit by hand
 # Generator token: 10BE3573-1514-4C36-9D1C-5A225CD40393
 
+c_pack <- function(root_obj) {
+    .Call('_RcppMsgPack_c_pack', PACKAGE = 'RcppMsgPack', root_obj)
```

Comment:
This may change when I re-run this.  With Rcpp 0.12.12 -- on CRAN for a few weeks now -- it should no longer generate a `.Call()` with `PACKAGE=`.  To be seen, also no worries.

Reply:
Turns out it was the missing `.registration=TRUE` in `NAMESPACE`.  No worries,